packages/conventional-changelog-core/test/test.js

```diff
@@ -492,6 +492,27 @@ describe('conventionalChangelogCore', function() {
     }));
   });
 
+  it('should generate correct compare url for bitbucket', function(done) {
+    preparing(18);
+
+    conventionalChangelogCore({
+      releaseCount: 0,
+      pkg: {
+        path: __dirname + '/fixtures/_bitbucket-host.json'
+      },
+      config: require('../../conventional-changelog-angular')
```

Comment:
Referenced the local changelog-angular

Reply:
Is this the only location in `conventional-changelog-core` that we test for a valid `compare` link?

Should this be extended to also test for GItLab, GitHub, and a unknown SCM platform?